### DISCUSSION TITLE
无法连接到安卓系统

### DISCUSSION THREAD

**Discussion by 2451760252** [Paving-Base/APK-Installer#325] [Q&A] [answered]
![image](https://github.com/Paving-Base/APK-Installer/assets/120788853/9c663bb7-375a-41ca-a26f-117a90f8eb26)
![image](https://github.com/Paving-Base/APK-Installer/assets/120788853/d7d6ace9-cfe2-45ed-b982-bc2d16321ad4)

**Comment by wherewhere** [comment, marked as answer]
重启一下，端口被占用了
